<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="3" rowspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QPushButton" name="complex_1_administra">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Administra Complex nº 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLCDNumber" name="complex_2_humitat">
        <property name="digitCount">
         <number>2</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="8" column="8" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>En creixement </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="aus_creixent_lcd">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>265</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="aus_creixent_progress_bar">
          <property name="value">
           <number>45</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0" colspan="11">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Informacio Adicional </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="8" colspan="3">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Complex nº 3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Humitat</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QLCDNumber" name="aus_totals">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>592</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Poblacio total de Aus </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Humitat</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="4" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Poblacio Adulta   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="altre_adults_lcd">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>230</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="altre_adults_progress_bar">
          <property name="value">
           <number>71</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="10">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Lluminositat</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="3">
       <widget class="QPushButton" name="complex_2_administra">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Administra Complex nº 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLCDNumber" name="complex_1_lluminositat">
        <property name="numDigits">
         <number>2</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>75</number>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Humitat</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLCDNumber" name="complex_2_lluminositat">
        <property name="numDigits">
         <number>2</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>95</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Poblacio total de ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="altre_total">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>321</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="9">
       <widget class="QLCDNumber" name="complex_3_humitat">
        <property name="digitCount">
         <number>2</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item row="3" column="8" colspan="3">
       <widget class="QPushButton" name="complex_3_administra">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Administra Complex nº 3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLCDNumber" name="complex_1_temperatura">
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="numDigits">
         <number>5</number>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>19.500000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="10" column="8" colspan="3">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>En creixement  </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="altre_creixent_lcd">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>91</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QProgressBar" name="altre_creixent_progress_bar">
          <property name="value">
           <number>29</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Lluminositat</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Complex nº 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLCDNumber" name="complex_2_temperatura">
        <property name="numDigits">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>21</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Temperatura</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="complex_1_humitat">
        <property name="numDigits">
         <number>2</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>17</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Temperatura </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="3">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Complex nº 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="11">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="10">
       <widget class="QLCDNumber" name="complex_3_luminositat">
        <property name="digitCount">
         <number>2</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Lluminositat</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Temperatura</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="4" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Poblacio Adulta   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="aus_adults_lcd">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>327</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="aus_adults_progress_bar">
          <property name="value">
           <number>55</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" rowspan="2" colspan="11">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="11">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QLCDNumber" name="complex_3_temperaura">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>23</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCanvia_Valors">
    <property name="title">
     <string>Canvia Valors</string>
    </property>
    <addaction name="actionCanvia_els_valors_del_programa"/>
   </widget>
   <addaction name="menuCanvia_Valors"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAjuda_amb_el_programa">
   <property name="text">
    <string>Ajuda amb el programa</string>
   </property>
  </action>
  <action name="actionDetall_sobre_els_animals">
   <property name="text">
    <string>Detall sobre els animals</string>
   </property>
  </action>
  <action name="actionWifi_Farm_v0_1">
   <property name="text">
    <string>Wifi_Farm v0.1</string>
   </property>
  </action>
  <action name="actionContacta_amb_l_empresa_fabricant">
   <property name="text">
    <string>Contacta amb l'empresa fabricant</string>
   </property>
  </action>
  <action name="actionConecxio_per_IP">
   <property name="text">
    <string>Conecxio per IP</string>
   </property>
  </action>
  <action name="actionCanvia_els_valors_del_programa">
   <property name="text">
    <string>Canvia els valors del programa</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>complex_1_administra</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>administra_complex_1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>complex_2_administra</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>administra_complex_2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>complex_3_administra</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>administra_complex_3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>844</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCanvia_els_valors_del_programa</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>finestra_canvia_valors()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>administra_complex_1()</slot>
  <slot>administra_complex_2()</slot>
  <slot>administra_complex_3()</slot>
  <slot>sortir()</slot>
  <slot>manual()</slot>
  <slot>finestra_canvia_valors()</slot>
 </slots>
</ui>
